src/test/groovy/tests/PrometheusTest.groovy

```diff
@@ -219,7 +191,7 @@ class PrometheusTest extends DataAnalyticsSteps{
     void rtDeniedActionByUsersTest() throws Exception {
         def query = "sum by (user) (increase(jfrog_rt_access{user!=\"UNKNOWN\", user!=\"_system_\", action_response=~\"DENIED.*\"}[10m]) > 0)"
```

Comment:
use `' '` and you won't need to use escapes here since you aren't using any interrpolation